parser_with_temp.py

```diff
@@ -0,0 +1,258 @@
+            table_page = BeautifulSoup(self.post(
+                self.base_url + route + self.token + page + str(
+                    page_number + 1)).text.encode('utf-8'), "html.parser")
+            table_body = table_page.find('table', {'class': 'list'}).find(
+                'tbody')
+            for row in table_body.findAll('tr'):
+                if row.find('input').get('value') != '':
+                    _id_list.append(int(row.find('input').get('value')))
+        return _id_list
+
+
+def create_summary_dictionary(session, order_id):
+    """
+    Requests a page with full table, using session and order id.
+    Calls filling_order_table, wich returns detailed data of ordered goods.
+    Grabs necessary data into the summary dictionary.
+    :param session: Session object
+    :param order_id: int - number of desired order.
+    :return: summarized dictionary with full data about the order
+    """
+    route = 'index.php?route=sale/order/info'
+    order = '&order_id='
+    order_url = session.base_url + route + session.token + order + str(
+        order_id)
+    with session.post(order_url) as full_table_page_response:
+        full_table_page = full_table_page_response.text
+        table = BeautifulSoup(
+            full_table_page.encode('utf-8'), 'html.parser').findAll(
+            'table', {'class': 'form'})
+        summary_dictionary = dict()
+        summary_dictionary['buyer'] = table[0].find(
+            'td', text=u'Покупець').next_sibling.next_sibling.string
+        summary_dictionary['email'] = table[0].find(
+            'td', text=u'E-mail:').next_sibling.next_sibling.string
+        summary_dictionary['phone'] = table[0].find(
+            'td', text=u'Телефон').next_sibling.next_sibling.string
+        summary_dictionary['city'] = table[1].find(
+            'td', text=u'Місто:').next_sibling.next_sibling.string
+        summary_dictionary['order_date'] = table[0].find(
+            'td', text=u'Дата замовлення:').next_sibling.next_sibling.string
+        summary_dictionary['sum'] = table[0].find(
+            'td', text=u'Усього:').next_sibling.next_sibling.string
+        summary_dictionary['summary_order_goods'] = filling_order_table(
+            full_table_page)
+    indexed_dictionary = {}
+    indexed_dictionary[order_id] = summary_dictionary
+    with open('temp.txt', 'a') as temp:
+        temp.write(str(indexed_dictionary) + '\n')
+
+
+def filling_order_table(full_table_page):
+    """
+    Returns a list of dictionaries with all ordered goods and their properties
+    in certain order.
+    :param full_table_page:
+    :return: list of dictionaries with all ordered goods and their properties.
+    """
+    product_table_list = BeautifulSoup(
+        full_table_page.encode('utf-8'), 'html.parser').find(
+        id='tab-product').find('tbody').findAll('tr')
+    orders = []
+    for row in product_table_list:
+        order = {'good': row.find('td').find('a').string,
```

Comment:
I don't expect don't use bs4. I expect to use BS4 together with xPath expressions.